tests/specs/task/dependencies/diamond_big.out

```diff
@@ -0,0 +1,12 @@
+Task e deno run e.js
+Running e
+[UNORDERED_START]
+Task b deno run b.js
+Running b
+Task d deno run d.js
```

Comment:
I'm thinking we might want to show that this task is being run because it's a dependency of another task. Could be done in a follow up though.